pallets/elections/src/lib.rs

```diff
@@ -87,11 +106,23 @@ pub mod pallet {
 
             Ok(())
         }
+
+        #[pallet::weight((T::BlockWeights::get().max_block, DispatchClass::Operational))]
+        pub fn change_reserved_members(
```

Comment:
tbh I was thinking about pallet tests likewise `pallet_aleph` has

Reply:
Tests will be added but first, we need to decouple some dependencies from this pallet to make it easier. Definitely not in this pr

Reply:
fair enough, is there any Jira for this?

Reply:
added as 981